man/fi_peer.3.md

```diff
@@ -286,6 +286,91 @@ on the local node.
    creates a reference between the peer_cq object and its own cq.
 ```
 
+# PEER COUNTER
+
+The peer counter defines a mechanism by which a peer provider may increment value or error into
+the counter owned by another provider. This avoids the overhead of the libfabric user
+needing to access multiple counters.
+
+To setup a peer counter, a provider creates a fid_peer_cntr object, which links back
+to the provider’s actual fid_cntr. The fid_peer_cntr object is then
+imported by a peer provider. The fid_peer_cntr defines callbacks
+that the providers use to communicate with each other.
+The provider that allocates the fid_peer_cntr is known as the owner,
+ with the other provider referred to as the peer.
+An owner may setup peer relationships with multiple providers.
+
+Peer counters are configured by the owner calling the peer’s fi_cntr_open() call.
+ The owner passes in the FI_PEER flag to fi_cntr_open().
```

Comment:
nit: space at start of line

Reply:
this line is dropped after the simplification in new revision.